stripe-ui-core/src/main/java/com/stripe/android/uicore/image/NetworkImageDecoder.kt

```diff
@@ -34,6 +34,14 @@ class NetworkImageDecoder {
         }!!
     }
 
+    suspend fun decode(
+        url: String
+    ): Bitmap {
+        return BitmapFactory.Options().run {
+            decodeStream(url)
+        }!!
```

Comment:
Can we make this optional instead of force unwrapping? Same for the other method above. It seems risky to do this when loading a url, we have no control over its content and it can crash the app. Just not showing any image would be better.